Project-Aurora/Project-Aurora/Settings/Configuration.cs

```diff
@@ -395,6 +395,7 @@ public class Configuration : Settings
         public BitmapAccuracy BitmapAccuracy { get { return bitmapAccuracy; } set { bitmapAccuracy = value; InvokePropertyChanged(); } }
 
         public bool updates_check_on_start_up;
+        public bool pointer_updates_check_on_start_up;
```

Comment:
This should be setup as a property similar to BitmapAccuracy, not a variable like the rest of the old parts of Configuration